apis/cloud.redhat.com/v1alpha1/clowdjobinvocation_types.go

```diff
@@ -89,6 +89,12 @@ type IqeJobSpec struct {
 
 	// sets value passed to IQE 'IQE_PARALLEL_WORKER_COUNT' arg
 	ParallelWorkerCount string `json:"parallelWorkerCount,omitempty"`
+
+	// sets value passed to IQE 'IQE_RP_ARGS' report portal args
+	RpArgs string `json:"RpArgs,omitempty"`
```

Comment:
The JSON property name should use `camelCase`

```suggestion
	RpArgs string `json:"rpArgs,omitempty"`
```